Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [workspace]
 members = [
+    "fortanix-vme/aws-nitro/eif-tools",
```

Comment:
It's nitro enclaves. Nitro is something else.

Reply:
Yes you have a point that `aws-nitro-enclaves` is a more suitable name.